actions/dependabot-gar-login/action.yaml

```diff
@@ -0,0 +1,44 @@
+name: "Update Google Artifact Registry Token"
+description: "Updates the token for Google Artifact Registry and sets it as a secret for Dependabot at the repository level."
+
+runs:
+  using: "composite"
+  steps:
+    - id: get-secrets
+      uses: grafana/shared-workflows/actions/get-vault-secrets@main
```

Comment:
This needs to refer to the action locally. See the top-level readme for how to do this